Keyed/Messages.xml

```diff
@@ -47,37 +47,37 @@
 
   <MessageCriticalAlert>ВНИМАНИЕ: {0}</MessageCriticalAlert>
 
-  <MessageMustDesignateHarvestable>Нужно указать достаточно созревшие собираемые растения.</MessageMustDesignateHarvestable>
+  <MessageMustDesignateHarvestable>Нужно указать созревшие собираемые растения.</MessageMustDesignateHarvestable>
   <MessageMustDesignateHarvestableWood>Нужно указать достаточно выросшие растения.</MessageMustDesignateHarvestableWood>
   <MessageMustDesignatePlants>Нужно указать растения.</MessageMustDesignatePlants>
-  <MessageMustDesignateHaulable>Нужно указать предметы, которые можно переносить. Некоторые предметы всегда можно переносить, в этом случае указать их для переноски будет невозможно.</MessageMustDesignateHaulable>
+  <MessageMustDesignateHaulable>Нужно указать предметы, которые можно переносить. Некоторые предметы можно переносить всегда, их нельзя указать.</MessageMustDesignateHaulable>
   <MessageMustDesignateMineable>Нужно указать непроходимую добываемую породу.</MessageMustDesignateMineable>
-  <MessageMustDesignateHuntable>Нужно указать животных, пригодных для охоты.</MessageMustDesignateHuntable>
+  <MessageMustDesignateHuntable>Нужно указать животных, на которых возможно охотиться.</MessageMustDesignateHuntable>
   <MessageMustDesignateTameable>Нужно указать приручаемых животных.</MessageMustDesignateTameable>
   <MessageMustDesignateClaimable>Нужно указать древние заброшенные сооружения.</MessageMustDesignateClaimable>
   <MessageMustDesignateSmoothableFloor>Нужно указать неровную каменную поверхность.</MessageMustDesignateSmoothableFloor>
-  <MessageNothingCanRemoveThickRoofs>Толстые каменные потолки, удалить невозможно.</MessageNothingCanRemoveThickRoofs>
+  <MessageNothingCanRemoveThickRoofs>Потолок в виде горной породы удалить невозможно.</MessageNothingCanRemoveThickRoofs>
   <MessageAlreadyInStorage>Находится на хранении, по мере необходимости будет перемещено в первую очередь.</MessageAlreadyInStorage>
-  <MessageMustDesignateStrippable>Отличать трупы от раненых.</MessageMustDesignateStrippable>
+  <MessageMustDesignateStrippable>Нужно указать мертвых или тяжело раненных, с которых можно снять одежду.</MessageMustDesignateStrippable>
```

Comment:
https://ru.wiktionary.org/wiki/%D1%82%D1%8F%D0%B6%D0%B5%D0%BB%D0%BE%D1%80%D0%B0%D0%BD%D0%B5%D0%BD%D1%8B%D0%B9